server/parsec/components/postgresql/shamir.py

```diff
@@ -0,0 +1,8 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+from parsec.components.shamir import BaseShamirComponent
+
+
+class PGShamirComponent(BaseShamirComponent):
+    # TODO
```

Comment:
```suggestion
    # TODO: https://github.com/Scille/parsec-cloud/issues/7358
```

We have a specific issue to track that, When we can, it's nice to reference it in the `TODO` & co